package.json

```diff
@@ -479,10 +479,10 @@
 	  "SLDS-component-path": "/components/split-view",
 	  "dependencies": [
 		{
-		  "component": "header"
+		  "component": "split-view-header"
```

Comment:
Let's go back to `header` and `list`. `import form 'split-view/split-view-header'` is repetitive and inconsistent with rest of library. I need to make sure I look at all the file names in my reviews better. I usually forget that part and focus on the code.